File: SysGestionTurismo/src/main/java/pe/edu/upeu/sysgestionturismo/modelo/Cliente.java
package pe.edu.upeu.sysgestionturismo.modelo;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
public class Cliente {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long idCliente;

    private String nombres;
    private String apellidos;
    private String numDocumento;
    private String imagenPerfil;
    private String whatsappContacto;
    private String correo;
    private String direccion;
    private String tipoDocumento;
}

File: SysTurismoJPC/app/src/main/java/pe/edu/upeu/systurismojpc/modelo/Cliente.kt
package pe.edu.upeu.systurismojpc.modelo

data class ClienteDto(
    var idCliente: Long,
    var nombreCompleto: String,
    var correo: String,
    var telefono: String,
    var direccion: String
)

data class ClienteResp(
    val idCliente: Long,
    val nombreCompleto: String,
    val correo: String,
    val telefono: String,
    val direccion: String
)

fun ClienteResp.toDto(): ClienteDto {
    return ClienteDto(
        idCliente = this.idCliente,
        nombreCompleto = this.nombreCompleto,
        correo = this.correo,
        telefono = this.telefono,
        direccion = this.direccion
    )
}
